app/Http/Controllers/NumbersController.php

```diff
@@ -18,12 +18,18 @@ public function create()
 
     public function store(Request $request)
     {
-        $this->validate($request, [
+        $number = preg_replace('/[^\d]/', '', $request->input('number'));
```

Comment:
Nice work here. Wondering if there's any way to clean this up, but TBH, i can't think of anything, so guessing this is probably the best it can be.